builder/docker/docker.go

```diff
@@ -72,6 +72,8 @@ func Start(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
 		"192.168.0.0/16",
 		"--insecure-registry",
 		"100.64.0.0/10",
+                "--exec-opt",
```

Comment:
Done.

Note that this change probably was unneeded anyway, but I left it in to be explicit. If docker decides to changes their default it wont't be impacted (although obviously docker in docker does not get upgraded automatically so this probably won't happen)